tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py

```diff
@@ -153,33 +146,31 @@ def fluxes_from_temperature_full_domain(F, V, k):
 F_known_u = u_np1 * v / dt * dx + alpha * dot(grad(u_np1), grad(v)) * dx - u_n * v / dt * dx
 u_np1.rename("T", "")
 t = 0
-u_D.t = t + coupling._precice_tau
-assert (dt == coupling._precice_tau)
+u_D.t = t + precice._precice_tau
+assert (dt == precice._precice_tau)
 
 file_out = File("Solid/VTK/%s.pvd" % solver_name)
 
-while coupling.is_coupling_ongoing():
+while precice.is_coupling_ongoing():
 
     # Compute solution
     solve(a == L, u_np1, bcs)
 
     # Dirichlet problem obtains flux from solution and sends flux on boundary to Neumann problem
     fluxes = fluxes_from_temperature_full_domain(F_known_u, V, k)
```

Comment:
Could this line also go into the adapter?

Reply:
see https://github.com/precice/fenics-adapter/issues/3, if you mean line 167